<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="MSTSYS0098" name="com.mst.automationtraining.executionpage.TestExecution" tests="2" failures="0" timestamp="26 Dec 2018 11:02:01 GMT" time="76.816" errors="1">
  <testcase name="form101Submission" time="69.486" classname="com.mst.automationtraining.executionpage.TestExecution"/>
  <testcase name="afterSuite" time="7.330" classname="com.mst.automationtraining.executionpage.TestExecution">
    <error type="java.lang.RuntimeException" message="javax.mail.AuthenticationFailedException: 535-5.7.8 Username and Password not accepted. Learn more at
535 5.7.8  https://support.google.com/mail/?p=BadCredentials r187sm93010901pfc.63 - gsmtp
">
      <![CDATA[java.lang.RuntimeException: javax.mail.AuthenticationFailedException: 535-5.7.8 Username and Password not accepted. Learn more at
535 5.7.8  https://support.google.com/mail/?p=BadCredentials r187sm93010901pfc.63 - gsmtp

	at com.mst.automationtraining.mail.Mail_report.send_report(Mail_report.java:75)
	at com.mst.automationtraining.executionpage.TestExecution.afterSuite(TestExecution.java:180)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:350)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: javax.mail.AuthenticationFailedException: 535-5.7.8 Username and Password not accepted. Learn more at
535 5.7.8  https://support.google.com/mail/?p=BadCredentials r187sm93010901pfc.63 - gsmtp

	at com.sun.mail.smtp.SMTPTransport$Authenticator.authenticate(SMTPTransport.java:843)
	at com.sun.mail.smtp.SMTPTransport.authenticate(SMTPTransport.java:765)
	at com.sun.mail.smtp.SMTPTransport.protocolConnect(SMTPTransport.java:687)
	at javax.mail.Service.connect(Service.java:388)
	at javax.mail.Service.connect(Service.java:246)
	at javax.mail.Service.connect(Service.java:195)
	at javax.mail.Transport.send0(Transport.java:254)
	at javax.mail.Transport.send(Transport.java:124)
	at com.mst.automationtraining.mail.Mail_report.send_report(Mail_report.java:70)
	... 20 more
]]>
    </error>
  </testcase> <!-- afterSuite -->
</testsuite> <!-- com.mst.automationtraining.executionpage.TestExecution -->
